### ISSUE TITLE
False positive, expected ENDLOOP

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1563]
```abap
LOOP AT t_table_name INTO DATA(db_table_name).

      SELECT DISTINCT blah, blah_blah FROM (db_table_name) INTO @tree WHERE (t_where_clause_blah).
        SELECT SINGLE * FROM (db_tree_table_name) INTO @m_tree WHERE blah = @blah-blah AND
                                                                                                      blah = @blah-blah.
        IF sy-subrc <> 0.
          MESSAGE e008 WITH db_table_name INTO DATA(msg).
        ENDIF.
      ENDSELECT.

    ENDLOOP.
```

reproducible in playground

**Comment by larshp** [comment]
The SELECT DISTINCT should be recognized as a https://syntax.abaplint.org/?filter=select#/statement/SelectLoop instead of a https://syntax.abaplint.org/?filter=select#/statement/Select

![image](https://user-images.githubusercontent.com/5888506/100063607-c64ad400-2e31-11eb-8b2b-1c0b76852fef.png)

**Comment by larshp** [comment]
```abap
DATA db_table_name TYPE string.
DATA t_where_clause_blah TYPE string.
TYPES: BEGIN OF ty_type,
         blah      TYPE i,
         blah_blah TYPE i,
       END OF ty_type.
DATA tree TYPE ty_type.
SELECT DISTINCT blah, blah_blah FROM (db_table_name) INTO @tree WHERE (t_where_clause_blah).
ENDSELECT.
```

**Comment by larshp** [comment]
@FreHu try again, 2.60.10 published

**Comment by FreHu** [comment]
thanks :+1:
